os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -79,6 +80,13 @@ static void ns_setstatusbits(u32 NewState)
 	device_mutex_unlock(RT_DEV_LOCK_FLASH);
 }
 
+static int ns_printf(const char *input)
+{
+	int ret;
+	ret = sedbg("%s \n", input);
```

Comment:
please add `#include <debug.h>` for `sedbg`

Reply:
Done